### ISSUE TITLE
Contact Info Widget: add name and image attributes

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#8106]
Related PR: #7850

#### Steps to reproduce the issue

1. Add a Contact Info Widget to your sidebar, and fill in all fields.
2. Use Google's testing tool, https://search.google.com/structured-data/testing-tool/u/0/ , to check the site URL.

You will receive the following warnings:
<img width="630" alt="screen shot 2017-09-25 at 14 28 27" src="https://user-images.githubusercontent.com/3392497/30808305-ddb4e432-a1fd-11e7-97c3-afd603cc84d2.png">

To solve this issue, we would need to:
1. Add the `name` attribute, maybe using a new widget field named "business name"
2. Add the `image` attribute by adding a new field allowing one to add a business logo to the widget.

Reported in 892360-hc

**Comment by ikrum** [comment]
Looking forward to get support for this
TIA

**Comment by tonyzeoli** [comment]
This isssue is valid and needs to be addressed. All my client sites are getting errors from Search Console because of this issue and it needs to be addressed.

Google Stuctured Data Testing Tool is reporting that the Contact Info widget had
two errors. The first is a value for an image field and the second is a value
for a name field. However, the JetPack widget does not have either a name or
image setting, so I'm not sure how to fix this.

It's also reporting price range on the same line, which is odd.

Here is the URL of the page being tested:

https://savethegreatsouthbay.org/event/sayville-creek-defender-clean-up/

Here's a screenshot:
https://cloudup.com/ctWUSN_5zQA

What should I do here?

**Comment by jeherve** [comment]
@tonyzeoli There isn't much you can do here to immediately fix this on your site. In its Structured Data testing tool, Google chooses to require specific fields, and recommend others. They currently require 2 fields that do not exist in Jetpack's Contact Info widget (name and image), and recommend another ([priceRange](https://schema.org/priceRange)).

The Contact Info widget would consequently need to be expanded to include those new fields in the widget settings. Based on those fields, we can add the necessary Schema markup on the frontend, that would solve those errors in Google's testing tool.

The widget currently lives here:
https://github.com/Automattic/jetpack/blob/6aefc26364af0aaa9b950bdc64ba5020619eb8a2/modules/widgets/contact-info.php

**Comment by tonyzeoli** [comment]
@jeherve Okay, so my best course of action then is...to just wait. Is that correct?

**Comment by jeherve** [comment]
@tonyzeoli That's correct, unless you feel comfortable contributing a patch to Jetpack before we get to it. We'll happily review any contribution!

**Comment by brookewp** [comment]
Another report: hc-18492861

**Comment by kbrown9** [comment]
Another report: 3508970-zen

**Comment by eoinsav** [comment]
Reported in #26343339-hc
